modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/index.tsx

```diff
@@ -91,11 +91,30 @@ const CustomView: React.FC<React.HTMLAttributes<HTMLElement>> = () => {
 				}
 			);
 
+			const newObjectViewFilterColumns = objectViewFilterColumns.map(
+				(filterColumn: any) => {
+					const definition = JSON.parse(filterColumn.definition);
+					const filterType = filterColumn.filterType.toLowerCase();
+					const objectFieldName = filterColumn.objectFieldName;
+
+					return {
+						...filterColumn,
+						definition,
+						filterBy: objectFieldName,
+						filterType,
+						valueList: definition[filterType].map((item: any) => {
+							return {label: item, value: item};
+						}),
+					};
+				}
+			);
+
```

Comment:
Hey @naraandrade Good logic here! 

1. To maintain code consistency, could you move this logic to `context.tsx` in the `TYPES.ADD_OBJECT_FIELDS` function? There we set the state of all properties of all columns inside `object View`, we can set `newObjectViewFilterColumn` there too.

2. Can you add a `filterColumn` type inside the `map` like `TObjectViewFilterColumn`?

Reply:
@danvb112 Ok, I'll move it to the `context.tsx` :) Thanks